### ISSUE TITLE
Japanese pluralization issue

### ISSUE THREAD

**Issue by richtr** [i18next/i18next#971]
I added a few additional pluralization tests for Japanese to a fork of this repo @ https://github.com/i18next/i18next/compare/master...richtr:broken_japanese_pluralization

IIUC, these additional pluralization tests for Japanese (`ja`) should always pick up and use the `test_0` resource key (not `test` or `test_plural` for Japanese).

To run this branch with the additional tests:

```
$> git clone https://github.com/richtr/i18next i18next-richtr
$> cd i18next-richtr
$> git checkout broken_japanese_pluralization
$> npm install
$> npm run test
```

The result is as follows:

```
PhantomJS 2.1.1 (Mac OS X 0.0.0): Executed 345 of 345 (2 FAILED) (0.287 secs / 0.069 secs)
TOTAL: 2 FAILED, 343 SUCCESS


1) correctly translates for ["translation:test",{"count":1,"lng":"ja"}] args
     Translator translate() with plural
     expected 'test_en' to deeply equal 'test_ja'

2) correctly translates for ["translation:test",{"count":2,"lng":"ja"}] args
     Translator translate() with plural
     expected 'tests_en' to deeply equal 'test_ja'
```

Is this an error in this library or are we using pluralization incorrectly with Japanese somehow? 

Perhaps the documentation could be updated to include how this should work?

**Comment by richtr** [comment]
FYI, this is what the tool @ https://www.i18next.com/plurals.html#how-to-find-the-correct-plural-suffix reports that the suffix should be for Japanese `ja` pluralization.

**Comment by jamuhl** [comment]
should be fixed in i18next@10.0.1

**Comment by richtr** [comment]
Thanks for the quick fix! It looks like this commit should fix things: https://github.com/i18next/i18next/commit/4217efecaf6516328ad2f9ea2b3061df33a41cf4.

**Comment by jamuhl** [comment]
That commit is published in v10.0.1
